src/data/equipment.txt

```diff
@@ -890,6 +896,7 @@ tailored vest	100	none
 The Emperor's new shirt	10	none
 Thinknerd T-Shirt	10	none
 trench coat	200	Mus: 100
+trench coat	200	Mus: 100
```

Comment:
this is duplicated

Reply:
Neat - this change has uncovered that existing issue!